modules/apps/site/site-cms-site-initializer/test/js/main/dashboard/AllStructureTypesDropdown.test.tsx

```diff
@@ -130,56 +132,76 @@ describe('[CMS Dashboard] Components: AllStructureTypesDropdown', () => {
 		).toBeInTheDocument();
 	});
 
-	xit('search by a structure and returns a filtered result', async () => {
-		mockStructures([
-			{id: '01', name: 'structure 01'},
-			{id: '02', name: 'structure 02'},
-		]);
+	it('searches by structure name and returns a filtered result', async () => {
+		jest.useFakeTimers();
+
+		global.fetch = jest
+			.fn()
+			.mockResolvedValueOnce({
+				json: jest.fn().mockResolvedValue({
+					items: [
```

Comment:
hey @interaminense I think those tests will break when merged on master because my follow-up PR changed the  items strucutre. Please take a look  https://github.com/fmarcio/liferay-portal/blob/d31cdfd1c010bc21c648a89b0a276c9e76e5350b/modules/apps/site/site-cms-site-initializer/test/js/main/dashboard/AllStructureTypesDropdown.test.tsx#L100

now it's ``{
				id: '01',
				label: {en_US: 'structure 01'},
			},``